spec/acceptance/letsencrypt_plugin_dns_cloudflare_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'letsencrypt::plugin::dns_cloudflare' do
+  supported = { 'Debian' => true,
+                'RedHat' => true }
+  supported.default = false
+
+  context 'with defaults values' do
+    pp = <<-PUPPET
+      class { 'letsencrypt' :
+        email  => 'letsregister@example.com',
+        config => {
+          'server' => 'https://acme-staging-v02.api.letsencrypt.org/directory',
+        },
+      }
+      class { 'letsencrypt::plugin::dns_cloudflare':
+        api_token => 'dummy-cloudflare-api-token',
+      }
+    PUPPET
+
+    if supported[fact('os.family')]
```

Comment:
I wonder if it's easier to use this:
```suggestion
    if ['Debian', 'RedHat'].include?(fact('os.family'))
```

Then you can drop the whole supported variable.

Reply:
Will do 👍